### PR TITLE
Feature gh pages research and development workstream edit

### PR DESCRIPTION
**Pull request by mnallajerla**

### CODE REVIEW

**Review by jamesmrollins — APPROVED** [review]
Corrected some minor punctuation and line stacking issues.

**Comment by jamesmrollins** [comment]
Hi @mnallajerla - I reviewed these files.  See comment regarding line 12 of the progress_reports file.  What month is the annual NIH report due?

File: submit_vaco_and_vapahcs_progress_report.md [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
+5.  Put together first draft of report using last year's drafts.
+6.  Coordinate with PI to get approval and submit.
 
-- VA IIR
-- ART VACO January
-- DSMB update
-- VA Palo Alto RDIS (summer?) 
+[NIH R01](https://github.com/lzim/research/tree/master/r01)
+VA Palo Alto RDIS - February
+NIH Annual Progress Report -
```
[the remark this answers is not in the document]
Comment (mnallajerla) [reply submit_vaco_and_vapahcs_progress_report.md]: Hi @jamesmrollins  thank you for catching this! It should be 11/15

[the remark this answers is not in the document]
Comment (mnallajerla) [reply submit_vaco_and_vapahcs_progress_report.md]: I just made the changes and these edits are ready to merge. Thank you!

**Pull request merged**